Lifestyle/librelinkup.1m.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+# for homebrew installation add '-S PATH="${PATH}:/opt/homebrew/bin/python3"' between /env and python3
+
+# <xbar.title>LibreLinkUp Status</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Florian Schlund</xbar.author>
+# <xbar.author.github>FloSchl8</xbar.author.github>
+# <xbar.desc>Gets your GCM from LibreLinkUp: https://librelinkup.com/</xbar.desc>
+# <xbar.dependencies>python3, python requests</xbar.dependencies>
+
+import requests
+import sys
+
+# your LibreLibkUp login (this is NOT your LibreView Login)
+email = ""
+password = ""
+
+# your region/country
+# available: us, eu, de, fr, jp, ap, au, ae
+country = "eu"
+
+# used for ordering multiple patients
+# the value from this id will be displayed in the menu bar first, all others can be seen in the dropdown
+# all IDs will also be shown after the glucose values
+# if you have only one patient you can leave this blank
+first_patient = ""
```

Comment:
These fit the use case of [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html).

Reply:
done :)